client/components/Supplier/Supplier.scss

```diff
@@ -1,3 +1,122 @@
 .supplier-title {
   color: blue;
+}
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: montserrat, sans-serif;
+}
+input, button {
```

Comment:
This will match *all* the input & button tags which will likely create unexpected behavior in the future when you add more input/button
I'd recommend create a unique classname for this target element